installer/genesis/alphanet/genesis.json

```diff
@@ -0,0 +1,143 @@
+{
```

Comment:
We may want to blow up and re-create alphanet using the betanet recipe, if we're ready for it.